packages/server/src/users/database/UserSchema.ts

```diff
@@ -1,15 +1,80 @@
-import { Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Prop, raw, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { Exclude, Expose } from 'class-transformer';
 import { ObjectId } from 'mongodb';
-import { Document } from 'mongoose';
+import mongoose, { Document } from 'mongoose';
```

Comment:
Chyba nie trzeba importować całego mongoose, wydaje mi się, że samo `ObjectId` działa (zamiast `mongoose.Schema.Types.ObjectId`), ale nie jestem tego pewna

Reply:
Powinno tak być, według kodu `export type ObjectId = Schema.Types.ObjectId;` także dokładnie skrót na to 👍

Reply:
to co zaproponowałeś nie działa, eslint mi nie pozwala na takie rzeczy.  Jak nie ma moongose to jest bląd, że ze Schema nie może korzystać.

Reply:
`import { ObjectId } from 'mongodb';` nie działa?

Reply:
Aaa, Tomkowi chyba chodziło o to, że  importując `ObjectId` uzyskamy ten sam efekt co korzystając z `mongoose.Schema.Types.ObjectId` (___bo___ `export type ObjectId = Schema.Types.ObjectId;`) , a tej linii nie trzeba nigdzie dopisywać 😀

Reply:
okej, teraz jasne xd